locksmith/src/fulfillment/dispatcher.ts

```diff
@@ -1,5 +1,6 @@
-var UnlockJs = require('@unlock-protocol/unlock-js')
-var HDWalletProvider = require('truffle-hdwallet-provider')
+import UnlockJs from '@unlock-protocol/unlock-js'
+
+const HDWalletProvider = require('truffle-hdwallet-provider')
```

Comment:
https://docs.ethers.io/ethers.js/html/api-advanced.html#hdnode in case this become useful

Reply:
we could easily expose this aspect of ethers